README.md

```diff
@@ -0,0 +1,288 @@
+                                                          КП ППвИС(часть 1)
+                                                             Установка
+
+Для выполнения первой части КП по курсу ППвИС разработан пример реализации волнового алгоритма поиска минимального пути в графе на базе библиотеки sc-memory.
+
+На вашем компьютере должна быть установлена версия ostis ветки 0.5.0 из репозитория: https://github.com/ShunkevichDV/ostis и Clion.
+
+Последние версии указанного примера и данного документа могут быть найдены в репозитории: https://github.com/ostis-apps/wave_find_path_sc_memory
+или на сервере: Info/Studinfo/~Методическое обеспечение кафедры/~Учебные курсы/2 курс/ПпвИС/1sem/КП 2020
+
+1) Запустите терминал и перейдите в папку ostis, выполните команду git clone https://github.com/ostis-apps/wave_find_path_sc_memory
+
+2) Перейдите в папку wave_find_path_sc_memory и скопируйте папку graph в ostis/kb
+
+3) Запустите Clion. Запустив среду, необходимо выбрать пункт меню Open Project и в появившемся диалоговом окне указать файл CMakeLists.txt,
+находящийся в папке с примером.
+
+4) После этого проект будет виден во вкладке Редактор(Edit). Перейдите в файл wavefindpath.cpp
+
+5) Измените переменные в функции main
```

Comment:
Can we use relative path and drop this point?

Reply:
no, need to use absolut path